db/seeds.rb

```diff
@@ -1,23 +1,52 @@
 # frozen_string_literal: true
 
-if Group.count.zero?
-  Group.create(
-    name: 'San Mateo County',
-    slug: 'smc',
-    domain: 'herokuapp.com',
-    group_type: :county,
-  )
-  Group.create(
-    name: 'San Jose',
-    slug: 'sj',
-    domain: 'localhost',
-  )
+def parse_from_json_file(filename)
+  json_file_str = File.read "lib/generators/dahlia_data_models/install/seed_data/#{filename}"
```

Comment:
@callaghanc I'm not sure how this file path works as part of the admin site? It seems like it's referencing a file within the gem repo?

Reply:
@jaredcwhite updated in latest addition to this PR